packages/react/src/context/slash-id-context.tsx

```diff
@@ -200,7 +267,7 @@ export const SlashIDProvider = ({
         __switchOrganizationInContext({ oid: newUser.oid });
       }
     },
-    [state, __switchOrganizationInContext, oid]
+    [state, __switchOrganizationInContext, oid, STORAGE_TOKEN_KEY]
```

Comment:
Why is `STORAGE_TOKEN_KEY` a dependency here?

Reply:
I understand why now, got confused with all caps. Probably better to use camelcase